articles/channel.html

```diff
@@ -574,12 +575,12 @@ <h3>Detailed Explanations for Channel Operations</h3>
 </li>
 <li>
 	If the channel is buffered, then at any time,
-	one of its sending goroutine queue and receiving goroutine queue must be empty.
+	at least one queue must be empty (out of the sending goroutine queue, the value buffer queue, and the receiving goroutine queue).

```

Comment:
Sorry, it looks I didn't explain this clearly in the last round of comments. Here, the value buffer queue should be totally uninvolved. Is the following one better?


	at least one goroutine queue of the channel must be empty.

Reply:
OK, I see what you mean now.

Reply:
Your original comment was fine btw, I just didn't read it closely enough :x